packages/features/portfolio/src/containers/portfolio-holdings/use-detailed-holdings.tsx

```diff
@@ -106,9 +117,10 @@ export function useDetailedHoldingsTable() {
           const isDebt = b.isNegative();
           const { icon, formattedTitle, titleWithMaturity, title } =
             formatTokenType(b.token, isDebt);
-          // const pointsPerDay = getPointsPerDay(b);
-          // const totalPoints =
-          //   arbPoints?.find(({ token }) => token === b.tokenId)?.points || 0;
+          const isBoostedToken = checkBoostedToken(
```

Comment:
Can we call this `isStarterBoost` instead and `checkStarterBoostToken` or something like that